fdbserver/fdbserver.actor.cpp

```diff
@@ -701,6 +702,7 @@ static void printUsage(const char* name, bool devhelp) {
 	printOptionUsage("--blob-credentials FILE",
 	                 "File containing blob credentials in JSON format. Can be specified "
 	                 "multiple times for multiple files. See fdbbackup usage for more details.");
+	printOptionUsage("--proxy MACHINE", "IP:port or host:port to proxy server for connecting to external network.");
```

Comment:
```suggestion
	printOptionUsage("--proxy PROXY:PORT", "IP:port or hostname:port to proxy server for connecting to external network.");
```

The name `machine` might be a bit misleading.